templates/csharp/command-and-response/appsettings.TestTool.json

```diff
@@ -1,15 +0,0 @@
-{
-  "Connections": {
```

Comment:
It seems the new *appsettings.Playground.json* is not created?

Reply:
Nice catch! I'll fix it in next PR considering the huge review list.